src/hooks/usePlanByEpg.ts

```diff
@@ -24,16 +24,63 @@ const usePlanByEpg = (channels: EpgChannel[]) => {
     ];
   }, [channels]);
 
+  // Theme configuration for the Planby EPG with only the colors set that are used
+  // Fixed values are used because of the tecnical dept we have with the current setup of the config colors values and the theme values
+  const theme = {
+    primary: {
+      600: '#141523',
+      900: '#141523',
+    },
+    grey: {
+      300: '#fff',
+    },
+    white: '#fff',
+    green: {
+      300: '#fff',
+    },
```

Comment:
Can we do this instead?

```suggestion
  const makeTheme = (primaryColor: string | undefined, backgroundColor: string | undefined) => {
    primary: {
      600: backgroundColor || '#141523',
      900: backgroundColor || '#141523',
    },
    grey: {
      300: primaryColor || '#fff',
    },
    white: primaryColor || '#fff',
    green: {
      300: primary color || '#fff',
    },
```

And within the component:

```tsx
  const theme = useMemo(() => makeTheme(config.styling.highlightColor, config.styling.backgroundColor), [config]);
```